### ISSUE TITLE
Disable multiline expression rule

### ISSUE THREAD

**Issue by thiago-you** [nbadal/ktlint-intellij-plugin#430]
Is there a way to disable this rule?

![Screenshot_199](https://github.com/nbadal/ktlint-intellij-plugin/assets/23344256/443e390d-17aa-4a05-b842-80b8ab32220c)

I tried this, but it didn't work.

## .editorconfig
ktlint_args=--disabled_rules=multiline-expression-wrapping
disabled_rules=multiline-expression-wrapping

**Comment by paul-dingemans** [comment]
Please consult ktlint documentation: https://pinterest.github.io/ktlint/latest/faq/#how-do-i-enable-or-disable-a-rule

**Comment by thiago-you** [comment]
Thanks for the quick response and the documentation link @paul-dingemans!

I updated the rule according to the documentation:
```
ktlint_standard_multiline-expression-wrapping = disabled
```
But now all rules seem to be disabled.

**Before applying the rule:**

![Screenshot_208](https://github.com/nbadal/ktlint-intellij-plugin/assets/23344256/15d98208-1301-40a9-8aa9-377ce84b0ad7)

**After applying the rule: (no visual warnings)**

![Screenshot_210](https://github.com/nbadal/ktlint-intellij-plugin/assets/23344256/9ac1b7fb-badd-40b3-a7b2-452c733517a5)

**Comment by paul-dingemans** [comment]
What is the content of your `ktlint-plugin.xml` file (located in `idea` folder)? Please also check wheter the `Problem Window` displays anything like below:
![Screenshot 2024-01-04 at 20 58 50](https://github.com/nbadal/ktlint-intellij-plugin/assets/5195292/f125fc82-b50d-4176-b4e7-c0c337de5d09)

**Comment by thiago-you** [comment]
It seems that is required to disable this rule as well:

```
ktlint_standard_string-template-indent
```

**This is the ktlint-plugin.xml:**

![Screenshot_212](https://github.com/nbadal/ktlint-intellij-plugin/assets/23344256/ee5b5e25-245a-42be-9a18-d86786ff0cfa)

By the way, I'm missing the configuration to show lint errors as a visual warning (instead of red errors), it is possible to config this into the XML file?

**Comment by paul-dingemans** [comment]
> It seems that is required to disable this rule as well:
> 
> ktlint_standard_string-template-indent

Yes indeed. This is shown in log and some popup-notification.
![Screenshot 2024-01-05 at 17 10 05](https://github.com/nbadal/ktlint-intellij-plugin/assets/5195292/47905835-5fb4-439a-8cf1-eaa52a6e1968)


> By the way, I'm missing the configuration to show lint errors as a visual warning (instead of red errors), it is possible to config this into the XML file?

No, but this is something that will be addressed in #414
